python/triton/compiler.py

```diff
@@ -895,10 +895,10 @@ def ttir_to_ttgir(mod, num_warps, num_stages, compute_capability):
     # The combine pass converts blocked layout to mma layout
     # for dot ops so that pipeline can get shared memory swizzled correctly.
     pm.add_triton_gpu_combine_pass(compute_capability)
-    pm.add_tritongpu_pipeline_pass(num_stages)
+    # pm.add_tritongpu_pipeline_pass(num_stages)
```

Comment:
Is disabling pipeline and prefetch pass expected here?

Reply:
It's not :D Thanks for catching it. Just re-enabled it; hopefully it doesn't make the tests fail again.